src/main/webapp/app/course/competencies/competency-accordion/competency-accordion.component.html

```diff
@@ -0,0 +1,80 @@
+<div>
+    <button
+        [id]="'competency-accordion-' + competency.id"
+        class="competency-accordion-header row align-items-center justify-content-between mt-2 position-relative flex-wrap"
+        (click)="toggle()"
+    >
+        <div class="col-auto d-md-block my-2">
+            <h3 class="fw-medium">
+                <fa-icon
+                    [icon]="getIcon(competency.taxonomy)"
+                    [fixedWidth]="true"
+                    [ngbTooltip]="'artemisApp.competency.taxonomies.' + (competency.taxonomy ?? 'none') | artemisTranslate"
+                    container="body"
+                />
+            </h3>
+        </div>
+        <div class="col-sm col py-2">
+            <h3>{{ competency.title }}</h3>
+        </div>
+        @if (!open) {
+            <div class="col-sm col py-2" style="max-width: 200px" [class.header-tooltip-open]="open">
+                <div class="row mb-2">
+                    <div class="col-auto d-md-block">
+                        <fa-icon [icon]="faList" [ngbTooltip]="'artemisApp.studentAnalyticsDashboard.competencyAccordion.exercises' | artemisTranslate" />
+                    </div>
+                    <div class="col">
+                        <jhi-student-analytics-dashboard-progress-bar [size]="'small'" [maxValue]="100" [currentValue]="exercisesProgress" [inline]="true" />
+                    </div>
+                </div>
+                <div class="row align-items-center justify-content-center">
+                    <div class="col-auto d-md-block">
+                        <fa-icon [icon]="faPdf" [ngbTooltip]="'artemisApp.studentAnalyticsDashboard.competencyAccordion.lectures' | artemisTranslate" />
+                    </div>
+                    <div class="col">
+                        <jhi-student-analytics-dashboard-progress-bar [size]="'small'" [maxValue]="100" [currentValue]="lectureUnitsProgress" [inline]="true" />
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm col text-end py-1 px-2 justify-content-center align-items-center" style="max-width: 75px">
+                <jhi-competency-rings [confidence]="this.confidence" [progress]="this.progress" [mastery]="this.mastery" [hideTooltip]="false" [playAnimation]="false" />
```

Comment:
What happens to the competency rings when we expand the accordion? 

Unexpanded:
<img width="931" alt="image" src="https://github.com/ls1intum/Artemis/assets/5898705/4bbf57a7-57c4-4b63-9dd5-b366a355ed55">

Expanded - rings are gone:
<img width="928" alt="Screenshot 2024-05-13 at 21 31 57" src="https://github.com/ls1intum/Artemis/assets/5898705/8606ad85-b46d-480e-b19d-43277597cf9c">

I think something does not make sense here.
What do you guys think of only showing the `View Lectures and Exercises` in the expanded section? Keeping the rest as in the unexpanded one.
<img width="284" alt="image" src="https://github.com/ls1intum/Artemis/assets/5898705/65dd0830-2b69-4061-aff6-b63f2d96e476">

@MaximilianAnzinger @bassner @kaancayli

Reply:
I have oriented myself fully to the Figma prototype that we have. If you have further suggestions I can implement them